dom/nodes/Document-createEvent.html

```diff
@@ -33,11 +33,23 @@
                   "isTrusted should be initialized to false");
   }, "createEvent('" + arg + "') should be initialized correctly.");
 }
-aliases.forEach(function(alias) {
-  testAlias(alias[0], alias[1]);
-  testAlias(alias[0].toLowerCase(), alias[1]);
-  testAlias(alias[0].toUpperCase(), alias[1]);
-});
+for (var alias in aliases) {
+  var iface = aliases[alias];
+  testAlias(alias, iface);
+  testAlias(alias.toLowerCase(), iface);
+  testAlias(alias.toUpperCase(), iface);
+
+  if (!alias.endsWith("s")) {
+    var plural = alias + "s";
+    if (!(plural in aliases)) {
+      test(function () {
+          assert_throws("NOT_SUPPORTED_ERR", function () {
+              var evt = document.createEvent(plural);
```

Comment:
Oh my, not just wrong, but mixed. I have informed Emacs and web-mode about the one true style.